apps/judicial-system/web/src/routes/CourtOfAppeal/Result/Result.tsx

```diff
@@ -20,10 +20,14 @@ import Conclusion from '@island.is/judicial-system-web/src/components/Conclusion
 
 import useAppealAlertBanner from '@island.is/judicial-system-web/src/utils/hooks/useAppealAlertBanner'
 import AppealConclusion from '@island.is/judicial-system-web/src/components/Conclusion/AppealConclusion'
+import { titleForCase } from '@island.is/judicial-system-web/src/utils/formHelper'
 
 import CaseFilesOverview from '../components/CaseFilesOverview/CaseFilesOverview'
 import CourtOfAppealCaseOverviewHeader from '../components/CaseOverviewHeader/CaseOverviewHeader'
-import { titleForCase } from '@island.is/judicial-system-web/src/utils/formHelper'
+
+import { useIntl } from 'react-intl'
```

Comment:
Please consider ordering the imports. Also, consider using a full path for the component imports above (not part of this PR but an opportunity for cleanup).